src/prefect/cli/kubernetes.py

```diff
@@ -26,7 +26,11 @@
 
 manifest_app = PrefectTyper(
     name="manifest",
-    help="Commands for generating Kubernetes manifests.",
+    help="Deprecated. Use 'helm server' instead.",
+    deprecated=True,
+    deprecated_name="prefect kubernetes",
+    deprecated_start_date="May 2023",
+    deprecated_help="Use 'helm server' instead."
```

Comment:
```suggestion
    deprecated_help="Use 'helm server' instead.",
```

In order to adhere to `black` formatting, this requires a trailing comma.

Reply:
Additionally note that `'helm server'` is not an acceptable suggestion per https://github.com/PrefectHQ/prefect/pull/9479#issuecomment-1558307826

Reply:
Ah yes, thanks, @madkinsz!

Reply:
@aryanv175 Are you interested in completing the updates here?